### PR TITLE
Return the language code for languages that are not supported

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@TrevorOctober ready for review

File: molo/core/content_import/helpers/locales.py [hunk 1]
```diff
@@ -4,6 +4,13 @@
 from unicore.content.models import Localisation
 
 
+def get_language(locale):
```
Comment (miltontony) [inline molo/core/content_import/helpers/locales.py]: Consider calling this `get_locale_english_name`

**Comment by TrevorOctober** [comment]
👍 once travis is happy

**Comment by miltontony** [comment]
:)

**Pull request merged**